### DISCUSSION TITLE
It did not work on my camera.

### DISCUSSION THREAD

**Discussion by Barbar16** [Arkady23/V380-Web#61] [Q&A] [answered]
Hello, I applied what you said, but I was not successful. It did not work on my camera.

Firmware HwV380E31_WF3_IPC60_ETH_20201218

[my camera](https://tr.aliexpress.com/item/1005004546622426.html?spm=a2g0o.order_list.order_list_main.5.37483d12s5lpAs&gatewayAdapt=glo2tur) 

![hata](https://github.com/Arkady23/V380-Web/assets/111146769/eeea4baf-7304-40fd-b0d9-68389c997d85)

**Comment by Arkady23** [comment]
It looks like the situation that the sd card is not formatted like fat32 or exfat. Try to reformat the card into a long one or replace it.

**Reply by Barbar16** [reply]
I have repeatedly formatted my SD card in both FAT32 and exFAT, but I got the same result each time.

===== quick check result dump =====
devID: 68.............
====== sensor ====>
======version ====>
[app_version]
name=AppV380E31_V1.1.7.2
date=20220728
[kernal_version]
name=KerV380E31_IPC60_ETH_V3.1.1
date=20201218
[hardware_version]
name=HwV380E31_WF3_IPC60_ETH
date=20201218
[base_version]
name=AppV380E31_V1.1.7.2
date=20220728
======cParams ====>
[CONST_PARAM]
ircut_flip=1
color_flip=0
firmware_type=67
network_reset_mode=4
uart2_usage=0
lens_type=2
starlight_mode=0
light_ctrl=0
night_with_color=1
reversal_value=0
ptz_opposite_run=0
ptz_dc_opposite_run=0
irfeed_source=1
resolution=1
force_res_level=0
vframe_encode_type=2
model_id= MVR33305
pano_auto_lens_type=0
oss_factoryid=11000100
standard_mode=1
panoY=540
panoRad=490
pano_crop_X=420
pano_Rad_res=697
pano_Y_res=768
pano_X_res=763
panoX=536
date_production=1684542239
time_zone=800
factory_id=1909170003
pcba_check_enable=0

[FRAME_QUANT_REC]
main_qp_max=43
main_qp_intra=39
main_qp_min=38
sub_qp_max=43
sub_qp_intra=39
sub_qp_min=38

[FRAME_QUANT_MOBILE]
main_qp_max=43
main_qp_intra=39
main_qp_min=38
sub_qp_max=43
sub_qp_intra=39
sub_qp_min=38

[FRAME_QUANT_ONVIF]
main_qp_max=43
main_qp_intra=39
main_qp_min=38
sub_qp_max=43
sub_qp_intra=39
sub_qp_min=38

======usb ====>
Bus 001 Device 002: ID 007a:8888
Bus 001 Device 001: ID 1d6b:0002
======network ====>
lo        Link encap:Local Loopback  
          inet addr:127.0.0.1  Mask:255.0.0.0
          UP LOOPBACK RUNNING  MTU:65536  Metric:1
          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
          TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1 
          RX bytes:0 (0.0 B)  TX bytes:0 (0.0 B)

domain tendawifi.com
nameserver 192.168.1.1
nameserver 8.8.8.8
Kernel IP routing table
Destination     Gateway         Genmask         Flags Metric Ref    Use Iface

Thank you for your prompt reply.

**Comment by Barbar16** [comment]
My SD card is 32GB I apologize, I have no knowledge of coding, and I am trying to learn by reading and experimenting. My IP camera is not useful in its current state without ONVIF support. I cannot use it without ONVIF support.

**Comment by Barbar16** [comment]
I think I should get myself a new camera. :(

**Reply by Arkady23** [reply, marked as answer]
I have a thought to double check the paths as described by dotvav here https://github.com/Arkady23/V380-Web/discussions/58#discussioncomment-6083566 and I'll try to understand.... maybe there really is something in this.

**Reply by Barbar16** [reply]
Thank you so much

**Reply by Arkady23** [reply]
As far as I understand dotvav, the dirname command does not work on his camera except for the tar command. I will remove dirname in the next fix (tomorrow).

**Comment by Barbar16** [comment]
I'm sorry, I don't know coding, but I wanted to add the error in the picture, hoping it might mean something to you.
![error](https://github.com/Arkady23/V380-Web/assets/111146769/1d80e1d1-0f87-438a-a7d2-aa8e1fa29608)
![err2](https://github.com/Arkady23/V380-Web/assets/111146769/dde3641e-0480-4eea-9891-5efa7884321b)

Could there be an issue related to the language?
Thank you very much for your interest and attention.

**Reply by Arkady23** [reply]
Please download yesterday's version::
https://github.com/Arkady23/V380-Web/releases/tag/Initial-installer

**Reply by Barbar16** [reply]
I tried, but the problem I have is still ongoing. I think I am doing something wrong.

**Reply by Arkady23** [reply]
This is probably such a camera. We need to deal with her.
